source/features/easy-toggle-files.tsx

```diff
@@ -20,7 +32,8 @@ async function init(): Promise<void> {
 void features.add(__filebasename, {
 	include: [
 		pageDetect.isPRFiles,
-		pageDetect.isPRCommit
+		pageDetect.isPRCommit,
```

Comment:
Shouldn't this also work in regular commits?
```suggestion
		pageDetect.isCommit,
```

Reply:
I don't why not!